smartMeter/ngsild-payloads/consumer.json

```diff
@@ -0,0 +1,128 @@
+{
+    "id": "urn:ngsi-ld:Consumer:01",
+    "type": "Consumer",
```

Comment:
The consumer should have a location (GPS or address)

Reply:
I have putted a location field in the Device. I put the same for the consumer?